src/main/java/mServer/crawler/sender/br/BrFilmDeserializer.java

```diff
@@ -0,0 +1,393 @@
+    if (!arrayItem.has(JSON_ELEMENT_NODE)) {
+      return Optional.empty();
+    }
+    
+    final JsonObject node = arrayItem.getAsJsonObject(JSON_ELEMENT_NODE);
+    if (!node.has(JSON_ELEMENT_START)) {
+      return Optional.empty();
+    }
+
+    return Optional.of(node.get(JSON_ELEMENT_START));
+  }
+
+  private Optional<JsonObject> getDetailClip(final JsonObject aViewer) {
+    if (!aViewer.has(JSON_ELEMENT_DETAIL_CLIP)) {
+      return Optional.empty();
+    }
+
+    return Optional.of(aViewer.getAsJsonObject(JSON_ELEMENT_DETAIL_CLIP));
+  }
+
+  private Optional<JsonArray> getVideoFileEdges(final JsonObject aViewer) {
+    if (!aViewer.has(JSON_ELEMENT_CLIP)) {
+      return Optional.empty();
+    }
+
+    final JsonObject clip = aViewer.getAsJsonObject(JSON_ELEMENT_CLIP);
+    if (!clip.has(JSON_ELEMENT_VIDEO_FILES)) {
+      return Optional.empty();
+    }
+
+    final JsonObject videoFiles = clip.getAsJsonObject(JSON_ELEMENT_VIDEO_FILES);
+    if (!videoFiles.has(JSON_ELEMENT_EDGES)) {
+      return Optional.empty();
+    }
+
+    return Optional.of(videoFiles.getAsJsonArray(JSON_ELEMENT_EDGES));
+  }
+
+  private Optional<JsonObject> getViewer(final JsonObject aBaseObject) {
+    if (!aBaseObject.has(JSON_ELEMENT_DATA)) {
+      return Optional.empty();
+    }
+
+    final JsonObject data = aBaseObject.getAsJsonObject(JSON_ELEMENT_DATA);
+    if (!data.has(JSON_ELEMENT_VIEWER)) {
+      return Optional.empty();
+    }
+
+    return Optional.of(data.getAsJsonObject(JSON_ELEMENT_VIEWER));
+  }
+
+  private Optional<BrUrlDTO> nodeToUrl(final JsonObject aNode) {
+    if (aNode.has(JSON_ELEMENT_PUBLIC_LOCATION)) {
+      if (aNode.has(JSON_ELEMENT_VIDEO_PROFILE)) {
+        final JsonObject videoProfile = aNode.getAsJsonObject(JSON_ELEMENT_VIDEO_PROFILE);
+        if (videoProfile.has(JSON_ELEMENT_ID)) {
+          if (videoProfile.has(JSON_ELEMENT_WIDTH)) {
+            if (!videoProfile.get(JSON_ELEMENT_WIDTH).isJsonNull()
+                && !videoProfile.get(JSON_ELEMENT_ID).isJsonNull()) {
+              return Optional.of(new BrUrlDTO(aNode.get(JSON_ELEMENT_PUBLIC_LOCATION).getAsString(),
+                  videoProfile.get(JSON_ELEMENT_WIDTH).getAsInt(),
+                  videoProfile.get(JSON_ELEMENT_ID).getAsString()));
+            }
+
+          } else {
+            printMissingDetails(JSON_ELEMENT_VIDEO_PROFILE + " -> " + JSON_ELEMENT_WIDTH);
+          }
+
+        } else {
+          printMissingDetails(JSON_ELEMENT_VIDEO_PROFILE + " -> " + JSON_ELEMENT_ID);
+        }
+
+      } else {
+        printMissingDetails(JSON_ELEMENT_VIDEO_PROFILE);
+      }
+
+    } else {
+      printMissingDetails(JSON_ELEMENT_PUBLIC_LOCATION);
+    }
+    return Optional.empty();
+  }
+
+  private void printMissingDetails(final String aMissingJsonElement) {
+    LOG.error(String.format(ERROR_MISSING_DETAIL_TEMPLATE, aMissingJsonElement));
+    //crawler.printMissingElementErrorMessage(aMissingJsonElement);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)